.github/actions/flatpak-builder-lint/action.yaml

```diff
@@ -0,0 +1,53 @@
+name: Run flatpak-builder-lint
+description: Runs flatpak-builder-lint with exceptions
+inputs:
+  artifactToLint:
+    description: Type of artifact to lint (builddir, repo, manifest)
+    required: true
+  path:
```

Comment:
`manifestPath`?

Reply:
No, the name needs cover repository and build directory cases.

Reply:
Why are you using the same variable for two different things then?

Reply:
Ah, I see. Understood.